core/main.go

```diff
@@ -36,6 +38,7 @@ func (c Core) Main() {
 	c.ConnectToVault()
 	c.Bind()
 	c.StartScheduler()
+	go c.metrics.RegisterBusEvents("*")
```

Comment:
I'm going to go out on a limb here and say that this doesn't compile any more?

Reply:
That was definitely a mistake on my part. It has been fixed now.